### PR TITLE
kubevirt: fix CLone VM Dialog namespaces

### PR DESCRIPTION
**Pull request by suomiy**
- always clone PVC and DataVolumes from VM namespace
- deprecate getResource and fix loading of Namespace model

@rawagner @mareklibra

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/src/components/modals/clone-vm-modal/clone-vm-modal.tsx [hunk 1]
```diff
@@ -233,24 +234,42 @@ export type CloneVMModalProps = CloneVMModalFirehoseProps &
...
-    getResource(VirtualMachineModel, { namespace, prop: 'virtualMachines' }),
+  const resources: FirehoseResource[] = [
+    {
+      kind: (useProjects ? ProjectModel : NamespaceModel).kind,
+      isList: true,
+      prop: 'namespaces',
+    },
+    {
+      kind: VirtualMachineModel.kind,
+      namespace,
```
Comment (rawagner) [inline frontend/packages/kubevirt-plugin/src/components/modals/clone-vm-modal/clone-vm-modal.tsx]: you want to get VMs from target namespace where user will be cloning the VM so you need to update the polling whenever the target namespace changes

Comment (suomiy) [reply frontend/packages/kubevirt-plugin/src/components/modals/clone-vm-modal/clone-vm-modal.tsx]: the namespace is updated accordingly

please see (few lines above)

```tsx
const [namespace, setNamespace] = React.useState(vmNamespace);
```

Comment (rawagner) [reply frontend/packages/kubevirt-plugin/src/components/modals/clone-vm-modal/clone-vm-modal.tsx]: sorry about that :) You're right

**Pull request merged**